### PR TITLE
foreman_openscap 0.5.3

### PR DESCRIPTION
**Pull request by shlomizadok**
Rails 4

### CODE REVIEW

File: rubygem-foreman_openscap/rubygem-foreman_openscap.spec [hunk 1]
```diff
@@ -14,12 +14,12 @@
 %global foreman_bundlerd_dir %{foreman_dir}/bundler.d
 
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.5.2
+Version: 0.5.3
 Release: 2%{?foremandist}%{?dist}
```
Comment (domcleal) [inline rubygem-foreman_openscap/rubygem-foreman_openscap.spec]: must be reset to 1

File: rubygem-foreman_openscap/rubygem-foreman_openscap.spec [hunk 2]
```diff
@@ -14,12 +14,12 @@
...
 Summary: Foreman plug-in for displaying OpenSCAP audit reports
 Group: Applications/System
 License: GPLv3
-URL: https://github.com/OpenSCAP/foreman_openscap
+URL: https://github.com/theforeman/foreman_openscap
```
Comment (domcleal) [inline rubygem-foreman_openscap/rubygem-foreman_openscap.spec]: This may be correct, but doesn't match the gemspec - please update it there too.

**Comment by shlomizadok** [comment]
Thanks.
Changed.
(I'll change the url on next release)

**Comment by domcleal** [comment]
Merged and building now, thanks @shlomizadok.

**Pull request merged**